quickstart/101-azure-api-management-create/main.tf

```diff
@@ -0,0 +1,25 @@
+resource "random_pet" "rg_name" {
+  prefix = var.resource_group_name_prefix
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = random_pet.rg_name.id
+  location = var.resource_group_location
+}
+
+resource "random_string" "name" {
```

Comment:
Should update the name to specify what this string is used for. "api_service_name" or something like that.

Reply:
Fixed